causal_testing/specification/causal_dag.py

```diff
@@ -465,5 +465,28 @@ def depends_on_outputs(self, node: Node, scenario: Scenario) -> bool:
             return True
         return any([self.depends_on_outputs(n, scenario) for n in self.graph.predecessors(node)])
 
+    def identification(self, base_test_case):
```

Comment:
Is there a reason why we need to pass in `base_test_case` rather than just the effect we want to identify (i.e. "total" or "direct")? Is it better practice for some reason or just a personal choice?

Reply:
The treatment and outcome `Variable`s are also stored in the `base_test_case` and are used in the identification function. 

I designed the `base_test_case` to hopefully store exactly what is required for identification

Reply:
Ah, that's sensible then